src/main/java/com/parasoft/findings/utils/doc/remote/RulesRestClient.java

```diff
@@ -126,6 +130,23 @@ public synchronized RuleInfo getRuleInfo(String sRuleId, String sAnalyzerId, Str
         }
     }
 
+    /**
+     * @param docUrl the value should be get from {@link #getRuleInfo(String, String)}
+     * @return content of given url, empty String if exception happens
+     */
+    public synchronized String getRuleContent(String docUrl) {
```

Comment:
RulesRestClient wraps a HttpsExecutor.ProxyExecutor. This ProxyExecutor has already handle the SSL problem(reger to https://github.com/parasoft/parasoft-findings-utils/blob/master/src/main/java/com/parasoft/findings/utils/doc/remote/HttpsExecutor.java#L68). So we can use RulesRestClient to get rule doc content from DTP.